### PR TITLE
2.2 unmanaged subnets

### PR DESCRIPTION
**Pull request by pmatulis**
Destined for master only.

### CODE REVIEW

File: en/installconfig-network-subnet-management.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+Title: Subnet Management | MAAS
+
+
+# Subnet Management 
+
+The topic of subnet management pertains to whether MAAS is in full control of a
```
Comment (evilnick) [inline en/installconfig-network-subnet-management.md]: I was hoping this would give a bit more introduction and explanation

Comment (pmatulis) [inline en/installconfig-network-subnet-management.md]: I can't think of anything else. Suggestions? Maybe I should scrub the introduction as there is really not much to say.

Comment (pmatulis) [inline en/installconfig-network-subnet-management.md]: @evilnick I added more stuff.

File: en/installconfig-network-subnet-management.md [hunk 2]
```diff
@@ -4,8 +4,13 @@ Title: Subnet Management | MAAS
 # Subnet Management 
 
 The topic of subnet management pertains to whether MAAS is in full control of a
-subnet or not. By default, a subnet is *managed* by MAAS but the administrator
-has the option of disabling this mode.
+subnet or not. When a subnet is *managed*, all aspects of IP address allocation
+is handled by MAAS. This includes DHCP leases and assigned static addresses.
```
Comment (evilnick) [inline en/installconfig-network-subnet-management.md]: are handled...

**Comment by pmatulis** [comment]
OK

**Pull request merged**